### PR TITLE
Use PostgreSQL in the CI

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates CI related files. Mainly, it updates CI to use PostgreSQL instead of Cassandra.

Sometimes, CI failed with timeout errors. And, basically, the runner (virtual machines) spec of GitHub Actions is fixed and we cannot scale up it.
https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources
(There is a [larger runners](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners) feature, but it is beta yet.)

So, I think we need to reduce resource usage in the CI. And, in my local environment testing, it seems that Cassandra's resource usage is slightly higher than PostgreSQL.

Therefore, I decided to use PostgreSQL instead of Cassandra in the CI.

Please take a look!

---

For your reference, after updating all files, I was able to run the `ct` command as follows.

```shell
$ ct install --config .github/ct-all.yaml
Installing charts...
Version increment checking disabled.
>>> helm version --short

------------------------------------------------------------------------------------------------------------------------
 Charts to be processed:
------------------------------------------------------------------------------------------------------------------------
 envoy => (version: "2.2.0", path: "charts/envoy")
 scalar-manager => (version: "1.0.0", path: "charts/scalar-manager")
 scalardb => (version: "2.5.0", path: "charts/scalardb")
 scalardb-graphql => (version: "1.3.0", path: "charts/scalardb-graphql")
 scalardl => (version: "4.5.1", path: "charts/scalardl")
 scalardl-audit => (version: "2.5.1", path: "charts/scalardl-audit")
 schema-loading => (version: "2.8.0", path: "charts/schema-loading")
------------------------------------------------------------------------------------------------------------------------

(snip)

------------------------------------------------------------------------------------------------------------------------
 ✔︎ envoy => (version: "2.2.0", path: "charts/envoy")
 ✔︎ scalar-manager => (version: "1.0.0", path: "charts/scalar-manager")
 ✔︎ scalardb => (version: "2.5.0", path: "charts/scalardb")
 ✔︎ scalardb-graphql => (version: "1.3.0", path: "charts/scalardb-graphql")
 ✔︎ scalardl => (version: "4.5.1", path: "charts/scalardl")
 ✔︎ scalardl-audit => (version: "2.5.1", path: "charts/scalardl-audit")
 ✔︎ schema-loading => (version: "2.8.0", path: "charts/schema-loading")
------------------------------------------------------------------------------------------------------------------------
All charts installed successfully
```

### CODE REVIEW

File: .github/postgresql.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+primary:
+  persistence:
+    enabled: false
+
+auth:
+  postgresPassword: postgres
```
Comment (kota2and3kan) [inline .github/postgresql.yaml:6]: This is a configuration file for [PostgreSQL packaged by Bitnami](https://github.com/bitnami/charts/tree/main/bitnami/postgresql).

File: .github/schema-loading.yaml [hunk 2]
```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
       - name: schema-loading
-        image: ghcr.io/scalar-labs/scalardb-schema-loader:3.5.2
+        image: ghcr.io/scalar-labs/scalardb-schema-loader:4.0.0-SNAPSHOT
```
Comment (kota2and3kan) [inline .github/schema-loading.yaml:42]: We are planning to use SNAPSHOT version containers in the `main` branch and CI in the future. So, I also use the SNAPSHOT version here.

File: charts/scalardb/ci/scalardb-ct-values.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+scalardb:
+  databaseProperties: |
+    scalar.db.storage=jdbc
+    scalar.db.contact_points=jdbc:postgresql://postgresql.default.svc.cluster.local:5432/postgres
+    scalar.db.username=postgres
+    scalar.db.password=postgres
```
Comment (kota2and3kan) [inline charts/scalardb/ci/scalardb-ct-values.yaml:6]: The `ct` command automatically uses `${ROOT_DIR_OF_CHART}/ci/*-values.yaml` files as custom values files when it test each chart. 
https://github.com/helm/chart-testing/blob/main/doc/ct_install.md

In this update, we have to configure `database.properties` to access PostgreSQL.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**